uiowa/uiowa
# Views block exposed filters

A replacement PR for #3829 that resolves #3804

# How to test
```
composer install
blt ds --site sandbox.uiowa.edu
```
## Article and people list blocks
* View some existing article and people lists block and compare the settings with the production sandbox site to make sure that the update hook worked as expected.
* Place blocks on a page and try changing the following settings, saving the block, saving the layout, and viewing the page.
  * Test setting headline or not
  * Test changing items to display
  * Test changing offset
  * Test setting pager
  * Confirm that the 3 settings above work predictably with one another
  * Test changing sort options
  * Test hiding fields.
  * Test adding a more link, setting a custom path, and setting custom text
  * Test setting filter values
  * Test exposing filters without setting a value
  * Test exposing filters with a value set
  * Test setting multiple filters at the same time
  * Test setting multiple filters to different variations of exposing the filter and not
  * Test that exposed filter and pager for one block do not affect any other views blocks on the same page of either the same type or a different types
  * Confirm that "Items per page" and "Offset" do not show up in exposed filters

### People list block
* When setting the "Person type" filter, it should limit the options display in "Person type status" drop-down to the same option.

## Event list block
```
drush @sandbox.local config:set config_split.config_split.event status true
drush @sandbox.local cim
```
* Create some local events.
* Run through the above tests.
